test/api/assignments_test.rb

```diff
@@ -29,8 +29,9 @@ def test_api_complains_if_no_key_is_submitted
   def test_api_accepts_submission_attempt
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
+        solution = { "THE CODE" => "ruby/one/code.rb" }
         post '/user/assignments',  { key: alice.key, code: 'THE CODE', path: 'ruby/one/code.rb', 
-          'language' => 'ruby','problem' => 'one' }.to_json
+          'language' => 'ruby','problem' => 'one', solution: solution }.to_json
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.